jep/4/README.adoc

```diff
@@ -194,24 +210,45 @@ and should consult the link:https://jenkins.io/event-calendar/[Jenkins Event
 Calendar] to make sure their meetings do not overlap with those already
 scheduled.
 
+[[chat]]
+==== Chats
+
+SIGs can *optionally* create dedicated channels for chats.
+These chats may be located in IRC, Gitter or other channels.
```

Comment:
Perhaps we could maintain a list of platforms that are acceptable.   We don't want official SIG chats hosted on obscure or add platforms.